File: backend/DataAccess/DaoGenerated/システム共有/tt_afsofusakiDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             送付先管理テーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afsofusakiDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afsofusaki";
		public string atenano { get; set; }                       //宛名番号
		public string riyomokuteki { get; set; }                  //利用目的
		public string adrs_sikucd { get; set; }                   //住所_市区町村コード
		public string adrs_tyoazacd { get; set; }                 //住所_町字コード
		public string adrs_todofuken { get; set; }                //住所_都道府県
		public string adrs_sikunm { get; set; }                   //住所_市区郡町村名
		public string? adrs_tyoaza { get; set; }                  //住所_町字
		public string? adrs_bantihyoki { get; set; }              //住所_番地号表記
		public string? adrs_katagaki { get; set; }                //住所_方書
		public string? adrs_yubin { get; set; }                   //住所_郵便番号
		public string sofusaki_simei { get; set; }                //送付先氏名
		public string? torokujiyu { get; set; }                   //登録事由
		public string? regsisyo { get; set; }                     //登録支所
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afsofusakiDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             送付先管理テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afsofusakiDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afsofusaki"
        Public Property atenano As String                       '宛名番号
        Public Property riyomokuteki As String                  '利用目的
        Public Property adrs_sikucd As String                   '住所_市区町村コード
        Public Property adrs_tyoazacd As String                 '住所_町字コード
        Public Property adrs_todofuken As String                '住所_都道府県
        Public Property adrs_sikunm As String                   '住所_市区郡町村名
        Public Property adrs_tyoaza As String                  '住所_町字
        Public Property adrs_bantihyoki As String              '住所_番地号表記
        Public Property adrs_katagaki As String                '住所_方書
        Public Property adrs_yubin As String                   '住所_郵便番号
        Public Property sofusaki_simei As String                '送付先氏名
        Public Property torokujiyu As String                   '登録事由
        Public Property regsisyo As String                     '登録支所
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
